packages/gatsby-source-wordpress/src/gatsby-node.js

```diff
@@ -138,7 +142,7 @@ exports.sourceNodes = async (
     touchNode,
     getNode,
     _auth,
-    reporter,
```

Comment:
We still should pass `reporter` to `normalize.downloadMediaFiles`

Reply:
Fixing that in https://github.com/gatsbyjs/gatsby/pull/16670

Not published yet anyway